app/uk/gov/hmrc/incorporatedentityidentificationfrontend/models/PageConfig.scala

```diff
@@ -21,8 +21,24 @@ import play.api.libs.json.{Json, OFormat}
 case class PageConfig(optServiceName: Option[String],
                       deskProServiceId: String,
                       signOutUrl: String,
-                      accessibilityUrl: String)
+                      accessibilityUrl: String,
+                      labels: Option[JourneyLabels] = None)
 
 object PageConfig {
+
+  def apply(optServiceName: Option[String],
+            optWelshServiceName: Option[String],
+            deskProServiceId: String,
+            signOutUrl: String,
+            accessibilityUrl: String) : PageConfig = {
+
+    val labels: Option[JourneyLabels] = optWelshServiceName match {
```

Comment:
This could be     val labels: Option[JourneyLabels] = optWelshServiceName.map(JourneyLabels(_))
... if you agree otherwise happy to accept it.